src/main/java/se/kth/sda/skeleton/user/User.java

```diff
@@ -16,9 +18,15 @@
     @OneToMany(mappedBy = "relatedUser", cascade = CascadeType.ALL)
     List<Post> createdPosts;
 
+    @OneToMany(mappedBy = "relatedUser", cascade = CascadeType.ALL)
+    List<Comment> createdComments;
+
     @OneToMany(mappedBy = "likedUser", cascade = CascadeType.ALL)
     List<PostLike> likedPosts;
 
+    @OneToMany(mappedBy = "likedUser", cascade = CascadeType.ALL)
+    List<PostLike> likedComments;
+
```

Comment:
Please remove this we will implement later

Reply:
it's been removed